pkg/controller/bundle/bundle_unpacker.go

```diff
@@ -7,6 +7,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/controller/security"
```

Comment:
nit: move this to the other OLM package imports that are grouped together.

Reply:
are there some gofmt settings I can use to to automagically sort them in the required way?

Reply:
I think goimportas actually handles this, which is why I opened #2798 when I was initially reviewing this PR. That linter configuration is already present in rukpak/deppy, and has helped cut down on the back-and-forth's for style-related PR review comments.